src/Components/GEOSldas_GridComp/Shared/LDAS_TileCoordType.F90

```diff
@@ -76,10 +76,15 @@ module LDAS_TileCoordType
      real    :: max_lat    ! maximum latitude (bounding box for tile)
      integer :: i_indg     ! i index (w.r.t. *global* grid that cuts tiles) 
      integer :: j_indg     ! j index (w.r.t. *global* grid that cuts tiles)
+     !if it is Cubed-Sphere grid, the index will be saved here for forcing
```

Comment:
I'm a bit confused by the naming of the field here. 
In my understanding the fields %i_indg and %j_indg are w.r.t. to the global grid that cuts tiles (regardless of whether that is a cube-sphere, EASE, or lat-long grid)
The second set of fields %cs_i_indg and %cs_j_indg should be w.r.t. to the (lat/long) "hash" grid that is used to quickly identify tiles that are within the lat/long grid cell(s) of the "hash" grid.  The name "cs_[x]_indg" is confusing, because the grid in question is NOT a cs grid.  How about something like:  %hash_i_indg and %hash_j_indg ???